build-ps/debian/percona-xtradb-cluster-server-5.7.mysql.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Start and stop the mysql (Percona XtraDB Cluster) daemon
+After=network.target
+After=syslog.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+User=mysql
+Group=mysql
+Type=forking
+TimeoutSec=0
+PermissionsStartOnly=true
+ExecStartPre=/usr/share/mysql/mysql-systemd-start pre
+ExecStart=/usr/sbin/mysqld --daemonize --pid-file=/var/run/mysqld/mysqld.pid $MYSQLD_OPTS $MYSQLD_RECOVER_START
+ExecStartPost=/usr/share/mysql/mysql-systemd-start post
+EnvironmentFile=-/etc/sysconfig/mysql
+LimitNOFILE = 5000
+Restart=on-failure
```

Comment:
I'm not sure if this is ok for PXC. When talking with Raghu long before he mentioned that for PXC automatic restarts are not good.
Also for `RestartPreventExitStatus=1` I have reported an upstream bug and it seems that the server is restarted at least several times even if this option is set, check this: https://bugs.mysql.com/bug.php?id=87210
So also check what @kbauskar  thinks about this.